runtime/src/lib.rs

```diff
@@ -106,8 +106,8 @@ pub mod opaque {
 
 /// This runtime version.
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("moonbase-alphanet"),
-	impl_name: create_runtime_str!("moonbase-alphanet"),
+	spec_name: create_runtime_str!("moonbeam"),
```

Comment:
@JoshOrndorff 
Will that impact the network (preventing to upgrade or ...) ? I'm not sure if the spec_name and impl_name are actually involved or just for description)..

Reply:
Good question. I don't know. I think we should make this change either way though. Then test it out. If it rules out runtime upgrades, maybe we just purge again?